src/player.cpp

```diff
@@ -2977,47 +3042,67 @@ Strength - 4;    Dexterity - 4;    Intelligence - 4;    Perception - 4"));
 
             werase(w_info);
             std::string s;
-            if (line == 0) {
+            if (line == 0) { //Torso
                 const int melee_roll_pen = std::max( -encumb( bp_torso ), -80 );
                 s += string_format( _("Melee attack rolls %+d%%; "), melee_roll_pen );
                 s += dodge_skill_text( - (encumb( bp_torso ) / 10));
                 s += swim_cost_text( (encumb( bp_torso ) / 10) * ( 80 - get_skill_level( skill_swimming ) * 3 ) );
                 s += melee_cost_text( encumb( bp_torso ) );
-            } else if (line == 1) { //Torso
+                s += thrown_cost_text( 2 * encumb( bp_torso ) );
+                s += fall_damage_penalty_text( calc_old_fall_damage_for_encumbrance_penalty( this, bp_torso ) * 100,
+                                               player::fall_damage_mod() * 100 );
```

Comment:
Alot of duplication here. These two lines occur three times and the only thing that varies is the `body_part`. `fall_damage_penalty_text()` should accept it as an argument.